x/bank/keeper/keeper_test.go

```diff
@@ -1473,12 +1473,27 @@ func (suite *KeeperTestSuite) TestMsgMultiSendEvents() {
 	require.Equal(abci.Event(event3), events[29])
 }
 
+// TestSpendableCoins verifies the behavior of SpendableCoins under various scenarios:
```

Comment:
@aljo242 I also added document on what testcases we have. I hope it cover every paths you need.